src/main/java/com/accesso/challengeladder/model/LoginForm.java

```diff
@@ -0,0 +1,24 @@
+package com.accesso.challengeladder.model;
```

Comment:
this class should be moved to the package 'com.accesso.challengeladder.requests' where the PostMatchesRequest is or the other way around, so we've all the form/request classes in the same package.